doc/source/getting_started/develop_pymapdl.rst

```diff
@@ -0,0 +1,337 @@
+
+.. _developing_pymapdl:
+
+===============
+Develop PyMAPDL
+===============
+
+You can help PyMAPDL library by fixing an bug or developing a new feature.
+To do that, you need to setup the repository in your local machine as it is explained
+in the following sections.
+
+
+Cloning the PyMAPDL repository
+==============================
+
+To clone PyMAPDL repository, you'll need to install a version control system (VCS) such as Git.
+Later run this code to clone the latest development version of PyMAPDL:
```

Comment:
```suggestion
Before cloning the PyMAPDL repository, you must install a version control system such as Git.
You can this run this code to clone the latest development version of PyMAPDL:
```
I removed the (VCS) because I don't see it being used later.